tests/func-tests/defaults_test.go

```diff
@@ -70,7 +70,6 @@ var _ = Describe("Check Default values", Label("defaults"), Serial, func() {
 			DeployKubeSecondaryDNS:           ptr.To(false),
 			DeployKubevirtIpamController:     ptr.To(false),
 			DeployTektonTaskResources:        ptr.To(false),
-			DeployVMConsoleProxy:             ptr.To(false),
```

Comment:
I think the functional tests fail, because of this change, but I'm not sure how to fix it.
I've tried running focused functional tests locally, but it outputs a long log, where it is hard to find relevant lines about test runs.

Reply:
I've kept the default value of `deployVMProxy` feature gate. It should fix the failing test.